src/main/Restaurant.java

```diff
@@ -14,4 +24,28 @@ public float getCash() {
     public void setCash(float cash) {
```

Comment:
Hey! This is @AndresVelandia, I think we should have a different approach regarding this method.
`public void setCash(float cash) {
    this.cash = cash;
}`

Create a setter method, and additionally add a method to add cash
`public void addCash(float cash) {
    this.cash += cash;
}`